hls4ml/converters/pytorch/core.py

```diff
@@ -50,7 +49,9 @@ def parse_activation_layer(operation, layer_name, input_names, input_shapes, nod
     # if layer['class_name'] != 'Activation':
     #    layer['activation'] = layer['class_name']
     if node.op == 'call_module':
-        if layer['class_name'] == 'ReLU' or layer['class_name'] == 'Sigmoid':
+        if layer['class_name'] in ['ReLU', 'Sigmoid', 'Tanh']:
+            if layer['class_name'] == 'Tanh':
+                layer['activation'] = 'tanh'
```

Comment:
We have a `layer['activation'] = layer['class_name']` a little bit above in the code, but for `tanh` we have to make it a lower case `t`. I guess that could be done just with an `layer['activation'] = layer['class_name'].lower()`, but I'm not sure if it's a general rule that the `activation` attribute is supposed to be lower case.

Reply:
I think it is safe. I see the templates will `lower()` it [here](https://github.com/fastmachinelearning/hls4ml/blob/main/hls4ml/backends/vivado/passes/core_templates.py#L183) so the only way to break if it is not lowered is for Quartus which does [this](https://github.com/fastmachinelearning/hls4ml/blob/main/hls4ml/backends/quartus/quartus_backend.py#L212) for god knows what reason. I think we generally expect it to be lowercase, so it should be fine to move it up to a line 45.

Reply:
Yeah, it's exactly because of this stuff in Quartus why I had to add this clause in the first place. But just making the attribute lower case works, so I changed it to that.